WaterLog_Backend/Controllers/SegmentEventsController.cs

```diff
@@ -154,7 +154,7 @@ public async Task<List<GetAlerts>> GetAlertsByPage(int id = 1)
                                 .FaultDate.
                                 Subtract(entry
                                 .AttendedDate)),
-                                ((entry.SensorType == EnumSensorType.WATER_FLOW_SENSOR) ? "Water Sensor" : "Sensor"),
+                                "Sensor",
```

Comment:
You are using this a lot along with faulty, consider constants

Reply:
Fixed